samples/highcharts/data-tools/datapool-html/demo.css

```diff
@@ -0,0 +1,65 @@
+:root {
+    --color-background: #fff;
+    --color-foreground: #000;
```

Comment:
IMO would be better to use --color-text, because of using for param `color:` that is applied for the font.